### ISSUE TITLE
copy-paste bug

### ISSUE THREAD

**Issue by terencehill** [gitextensions/gitextensions#4155]
Bug: if I copy some lines of a diff in the commit dialog including an initial line ending and paste them in my text editor I get an extra space added in the beginning of the pasted text (after the initial line ending).
It happens always regardless of the initial line ending being on a line that starts with +, - or space.

- GitExtensions version: 2.50.02
- OS version: Windows 8.1

Example, I select last 2 lines of this diff starting from the end of the first line
 
```
-	if(this.waterlevel)
-	if(this.goalcurrent==NULL)
+	if(this.waterlevel && !this.goalcurrent)
```
A wild space appears in the pasted text:
```
 
	if(this.goalcurrent==NULL)
	if(this.waterlevel && !this.goalcurrent)
```
Expected pasted text:
```

	if(this.goalcurrent==NULL)
	if(this.waterlevel && !this.goalcurrent)
```

**Comment by gerhardol** [comment]
I do not see the space. The printouts look the same, need to be protected from markdown. (Maybe the collaborators can see the raw text, I cannot.)

**Comment by RussKie** [comment]
That's what I can see:
![image](https://user-images.githubusercontent.com/4403806/33229646-acb41c3c-d226-11e7-8d30-d98ea354d6b2.png)

**Comment by RussKie** [comment]
I had some testing, and based on observations it seems that the problem may be caused by the way `ICSharpCode.TextEditor.TextEditorControl` is rendering new lines:

![image](https://user-images.githubusercontent.com/4403806/33229945-6b9d9eba-d22d-11e7-9ecc-324c587f46d2.png)

**Comment by terencehill** [comment]
This bug appears to be fixed in 3.5.2.

Did 718c84618 fix it?

- Git Extensions 3.5.2.12435
- Build 77413699a9c25b160a6a622311529372b5d87f5b
- Git 2.29.2 (recommended: 2.30.0 or later)
- Microsoft Windows NT 10.0.15063.0
- .NET Framework 4.7.2098.0
- DPI 96dpi (no scaling)

**Comment by RussKie** [comment]
Presume fixed
